taxonomies/ingredients.txt

```diff
@@ -5071,17 +5081,6 @@ ru:бифидобактерии-actiregularis, бифидобактерии acti
 es:bifidobacterias y otros fermentos lácticos
 fr:ferments lactiques dont bifidobacterium
 
-<en:ferments
```

Comment:
Those entries need to be added back. It looks like the translations were added on a slightly old version of the taxonomy.